integreat_cms/cms/models/external_calendars/external_calendar.py

```diff
@@ -31,6 +31,28 @@ class ExternalCalendar(AbstractBaseModel):
         ),
     )
     errors = models.CharField(verbose_name=_("import errors"), default="", blank=True)
+    created_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="created_by",
+        verbose_name=_("creator"),
+        help_text=_("The account that created this external calendar."),
+    )
+    last_changed_by = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="last_changed_by",
+        verbose_name=_("last changed by"),
+        help_text=_("The account that was the last to change this external calendar."),
+    )
+    last_changed_on = models.DateTimeField(
+        auto_now_add=True,
```

Comment:
```suggestion
        auto_now=True,
```
Using `auto_now` will make sure that this timestamp is automatically updated on save. `auto_now_add` will only set it on creation: https://docs.djangoproject.com/en/dev/ref/models/fields/#django.db.models.DateField.auto_now_add